modules/nf-core/mageck/count/tests/main.nf.test

```diff
@@ -0,0 +1,69 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
+// nf-core modules test mageck/count
+nextflow_process {
+
+    name "Test Process MAGECK_COUNT"
+    script "../main.nf"
+    process "MAGECK_COUNT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mageck"
+    tag "mageck/count"
+
+    test("mageck-count - fastq") {
+
+        config "./nextflow.config"
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test,test2', single_end:true] , // meta map
+                [file(params.test_data['mus_musculus']['illumina']['test_1_fastq_gz'],checkIfExists: true),
+                file(params.test_data['mus_musculus']['illumina']['test_2_fastq_gz'], checkIfExists: true)]
+                ]
+                input[1] = file(params.test_data['mus_musculus']['csv']['library'], checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                            process.out.versions,
+                            file(process.out.count.get(0).get(1)).readLines()[0]).match() }
```

Comment:
```suggestion
                            file(process.out.count_norm.get(0).get(1)).readLines()[0]).match() }
```

Reply:
or can you check both?

Reply:
there's norm and count but not count_norm, i can add norm for sure as soon as the tests run

Reply:
yes I realized, sorry! I changed it in my commit